pkg/test/ginkgo/test_runner.go

```diff
@@ -363,15 +373,21 @@ func updateEnvVars(envs []string) []string {
 	// copied from provider.go
 	// TODO: add error handling, and maybe turn this into sharable helper?
 	config := &clusterdiscovery.ClusterConfiguration{}
-	clientConfig, _ := framework.LoadConfig(true)
-	clusterState, _ := clusterdiscovery.DiscoverClusterState(clientConfig)
-	if clusterState != nil {
-		config, _ = clusterdiscovery.LoadConfig(clusterState)
+	if clientConfig, err := framework.LoadConfig(true); err == nil {
```

Comment:
err was just ignored before but now that you have it and are checking it, would you want to log it?  Same for below.

Reply:
Letting this go for now